### DISCUSSION TITLE
Xiaomi Aqara Motion sensor - RTCGQ11LM - always detect movement

### DISCUSSION THREAD

**Discussion by vhristev** [Koenkk/zigbee2mqtt#5680] [Devices]
WIthout any intergation today i upgrade my Zigbee2mqtt and my Home assistant and my motion sensors start detecting movement in the bathroom without anyone there. I notice it because my Bathroom lamp was always On ( it must be on when there is a movement ). 

Zigbee2mqtt
Current version: 1.17.0.1 

Home Assistant:
2020.12.7

My Bathroom Xiaomi Aqara Motion sensor  always show occupancy:

Zigbee2mqtt log:
`Zigbee2MQTT:info  2021-01-10 20:32:11: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":3,"illuminance_lux":3,"linkquality":110,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:11: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":3,"illuminance_lux":3,"linkquality":110,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:16: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":3,"illuminance_lux":3,"linkquality":110,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:16: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":3,"illuminance_lux":3,"linkquality":107,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:21: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":0,"illuminance_lux":0,"linkquality":110,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:21: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":0,"illuminance_lux":0,"linkquality":110,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:26: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":0,"illuminance_lux":0,"linkquality":107,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:26: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":0,"illuminance_lux":0,"linkquality":110,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:32: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":0,"illuminance_lux":0,"linkquality":110,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:33: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":3,"illuminance_lux":3,"linkquality":107,"occupancy":true,"voltage":3005}'
Zigbee2MQTT:info  2021-01-10 20:32:33: MQTT publish: topic 'zigbee2mqtt/BathroomPir', payload '{"battery":100,"illuminance":3,"illuminance_lux":3,"linkquality":107,"occupancy":true,"voltage":3005}'`

Here is from MQTT check the timestamp every few seconds is detecting movement. 
<img width="918" alt="Screenshot_10_01_21__20_38" src="https://user-images.githubusercontent.com/3786050/104132098-e1db5100-5383-11eb-99f4-d68d5039699c.png">

What I tried:

Remove it from Zigbee2mqtt devices
Remove it from Home Assistant
Remove it from MQTT

Reset it a couple of times and did the same. 

Till now nothing even I remove it again add it with New name and case is the same its like broken and cannot stop showing there is occupancy.

P.S. I have couple of such sensors hardware hackerd ( 5 sec. hack ) - https://community.smartthings.com/t/making-xiaomi-motion-sensor-a-super-motion-sensor/139806/27

**Comment by Koenkk** [comment]
Can you provide the debug log of this?

To enable debug logging set in `configuration.yaml`:
```yaml
advanced:
  log_level: debug
```

**Reply by vhristev** [reply]
Hey Koenkk , 

Thank you for looking it. Here is the output:
https://gist.github.com/vhristev/328cc3cf7032254d0fcb4e722989a6a4#file-gistfile1-txt

or

https://pastebin.com/GHikQQdH

**Reply by Koenkk** [reply]
Looking at your logs your sensor is constantly sending occupancy (`Received Zigbee message from 'BathroomPir', type 'attributeReport', cluster 'msOccupancySensing'`). I'm not sure what causes this but I'm quite certain it cannot be solved from Z2M, perhaps the sensor is defect? I suggest you also try putting the sensor in a box and see if it still detects occupancy.

**Reply by vhristev** [reply]
Thank you for debugging. I tried your suggestion i put the device in a box with the same result. I also put it in the hall where is the other "hall sensor" and it act the same always reporting motion. I clean the board with flux and alcohol made some components tests everything looks ok. My initial assumption is now alight with yours "its time to let it go" . This sensor is inside the bath and maybe at some point all the moister did something which is not visible.

**Comment by Boia11** [comment]
I solve with "force remove", battery remove and after 10 sec insert it and pairing device

**Reply by vhristev** [reply]
It does not help for me. Removing the battery is one of the things i did at the beginning.

**Reply by Boia11** [reply]
Try to re-pairing device and after restart ZigBee2MQTT add-on

**Comment by Bond246** [comment]
Hello,

 I have the exact same problem. I've switched from Phoscon/deconz to Z2M where this sensor worked like charme and now with Z2M this sensor first was sending occupancy every 10sec but good lux vales.

 I tried some repairing which worked one time but then after about an hour it stopped sending anything so I did a new pairing.

Really annoying... 🙁
